Detections/AWSCloudTrail/AWS_ChangeToVPC.yaml

```diff
@@ -1,8 +1,9 @@
 ﻿id: 65360bb0-8986-4ade-a89d-af3cf44d28aa
-name: AWS - Changes to VPC settings
+name: Changes to Amazon VPC settings
 description: |
-  'This alert monitors changes to VPC settings such as new ACL entries and routes in route tables.
-  More information: https://medium.com/@GorillaStack/the-most-important-aws-cloudtrail-security-events-to-track-a5b9873f8255'
+  'This alert monitors changes to Amazon VPC (Virtual Private Cloud) settings such as new ACL entries and routes in route tables.
+  More information: https://medium.com/@GorillaStack/the-most-important-aws-cloudtrail-security-events-to-track-a5b9873f8255 
+  and https://aws.amazon.com/vpc/'
 severity: Low
 requiredDataConnectors:
  - connectorId: AWS
```

Comment:
missing tactics section?

Reply:
tactics is there, but there was not a good mapping for techniques.  with cloud logs it can be a bit of a challenge to find a good approximation.  happy for your thoughts here.

Reply:
Decided to do close approximation, when MITRE CLOUD matrix comes out, we can revisist